exercises/exercise-09/index.ts

```diff
@@ -71,8 +71,16 @@ type ApiResponse<T> = (
     }
 );
 
-function promisify(arg: unknown): unknown {
-    return null;
+function promisify<T>(fn: (callback: (arg: ApiResponse<T>) => void) => void): () => Promise<T> {
```

Comment:
Also unclear to me what work there is to do here if you don't do the "higher difficulty" challenge.